pythonx/synta/__init__.py

```diff
@@ -21,6 +21,23 @@ def build():
     dirname = os.path.dirname(vim.current.buffer.name)
     cwd = os.getcwd()
 
+    envs = None
+    if os.path.exists(os.path.join(cwd, "AndroidManifest.xml")):
+        toolchain = "/opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android21-clang"
+        if 'synta_android_toolchain' in vim.vars:
```

Comment:
https://github.com/kovetskiy/synta/blob/master/plugin/go.vim#L30
can we do it like here

Reply:
done